miraclelinux/meta-debian-extended
# diffutils: remove duplicated package

# Purpose of pull request

meta-debian and meta-debian-extended both provide diffutils that
have small difference. This difference has resolved[1] so it's
okay to remove diffutils from this repo.

[1]: https://github.com/meta-debian/meta-debian/pull/227


# Test
## How to test

Check diffutils is removed from meta-deian-extended layer.

```
$ bitbake-layers show-recipes | grep diffutils -A3
diffutils:
  meta-debian          3.7
  meta                 3.7
distcc:
```

Run following command from console

```
root@qemuarm64:~# ptest-runner diffutils
```

ptest should not finish with exit code 512.

## Test result

```
root@qemuarm64:~# ptest-runner diffutils
START: ptest-runner
2020-08-17T04:17
BEGIN: /usr/lib/diffutils/ptest
make: Entering directory '/usr/lib/diffutils/ptest/tests'
make[1]: Entering directory '/usr/lib/diffutils/ptest/tests'
PASS: basic
PASS: bignum
PASS: binary
brief-vs-stat-zero-kernel-lies: skipped test: stat --format %s does not
work
SKIP: brief-vs-stat-zero-kernel-lies
PASS: cmp
PASS: colliding-file-names
PASS: diff3
PASS: excess-slash
PASS: help-version
PASS: invalid-re
PASS: function-line-vs-leading-space
PASS: ignore-matching-lines
PASS: label-vs-func
[   45.363398] random: perl: uninitialized urandom read (4 bytes read)
XFAIL: large-subopt
PASS: new-file
PASS: no-dereference
PASS: no-newline-at-eof
PASS: stdin
PASS: strcoll-0-names
PASS: filename-quoting
strip-trailing-cr: skipped test: requires a working valgrind
SKIP: strip-trailing-cr
PASS: colors
============================================================================
Testsuite summary for GNU diffutils 3.7
============================================================================
============================================================================
make[1]: Leaving directory '/usr/lib/diffutils/ptest/tests'
make: Leaving directory '/usr/lib/diffutils/ptest/tests'
DURATION: 43
END: /usr/lib/diffutils/ptest
2020-08-17T04:18
STOP: ptest-runner
```
